x/authz/client/cli/tx.go

```diff
@@ -123,6 +123,10 @@ Examples:
 						return err
 					}
 
+					if spendLimit.GetDenomByIndex(0) != staking.DefaultParams().BondDenom {
```

Comment:
I don't think this should be the case?

Reply:
You cannot use `DefaultParams`. Also, it is assumed denom at index 0 is always the native staking tokeN?

Reply:
Agree, no reason to restrict this to only staking Denom and index based comparison is not a correct way to handle this.

Reply:
Oops, my bad. Should validate against staking denom but not using defaultparams